models/intel/text-recognition-0016/composite-model.yml

```diff
@@ -0,0 +1 @@
+task_type: token_recognition
```

Comment:
old text-recognition models (0012/0014) have type optical_character_recognition but new ones, 0015/0016 got this token_recognition. Is there any special reason for that difference?

Reply:
I just copypasted the 0015 model. Do not remember, if it was me who added this `token_recognition` type.

Reply:
try `git blame` :)

Reply:
Well, seems like it was me. I can change it to `optical_character_recognition` if it is needed.

Reply:
@morkovka1337 being consistent is always better, otherwise we easily fall into chaos

Reply:
should I also change the corresponding value in 0015th model?

Reply:
@morkovka1337 yes, please

Reply:
Done